### PR TITLE
US6 SonySWR21Importer, Review

### PR DESCRIPTION
**Pull request by lucasbuschlinger**
Implemented US #8, ready to be reviewed.

### CODE REVIEW

File: plugins/de/opendiabetes/vault/plugin/importer/medtronic/MedtronicImporter.java [hunk 1]
```diff
@@ -61,21 +64,61 @@ public MedtronicImporter(final PluginWrapper wrapper) {
 
         //TODO javadoc for all Patterns
```
Comment (ocastx) [inline plugins/de/opendiabetes/vault/plugin/importer/medtronic/MedtronicImporter.java]: Remove TODO

**Review by ocastx — CHANGES_REQUESTED** [review]
- [x] Werden korrekte Typen benutzt?
- [x] Methoden haben korrekte Sichtbarkeit
- [x] Variablen haben korrekte Sichtbarkeit
- [x] Keine Rechtschreibfehler (Schrift/Grammatik)
- [x] Code gut verständlich
- [x] Eindeutige Variablennamen
- [x] Dokumentation in Form von JavaDoc vorhanden
- [x] Keine logischen Fehler
- [x] Checkstyle keine (drastischen) Fehler
- [x] FindBugs keine (drastischen) Fehler

Looks good so far, just remove the one ToDo that was done by this PR and we should consider removing the HTML files from the repo as they don't have anything to do with the code.

**Comment by lucasbuschlinger** [comment]
Removed ToDo with latest commit.
Good idea with removing the reports, where else should we put those?

Otherwise ready for merge, @ocastx ?

**Review by ocastx — APPROVED** [review]

**Comment by lucasbuschlinger** [comment]
Done, merging into master, closing this and #8

**Pull request merged**